src/cascade/executor/epiviz_runner.py

```diff
@@ -67,40 +67,45 @@ def meas_bounds_to_stdev(df):
     return df.drop(["meas_lower", "meas_upper"], axis=1)
 
 
-def add_mortality_data(model_context, execution_context):
+def add_mortality_data(model_context, execution_context, sex_id):
     """
     Gets cause-specific mortality rate and adds that data as an ``mtspecific``
     measurement by appending it to the bundle. Uses ranges for ages and years.
     This doesn't determine point-data values.
     """
-    MATHLOG.debug(f"Creating a set of mtspecific observations from IHME CSMR database.")
     csmr = meas_bounds_to_stdev(
         age_groups_to_ranges(execution_context, get_cause_specific_mortality_data(execution_context))
     )
     csmr["measure"] = "mtspecific"
     csmr = csmr.rename(columns={"location_id": "node_id"})
-    model_context.input_data.observations = pd.concat([model_context.input_data.observations, csmr], ignore_index=True)
+    csmr = csmr.query(f"sex_id == @sex_id")
+    MATHLOG.debug(f"Creating a set of {csmr.shape[0]} mtspecific observations from IHME CSMR database.")
+    model_context.input_data.observations = pd.concat([model_context.input_data.observations, csmr],
+                                                      ignore_index=True, sort=True)
 
 
-def add_omega_constraint(model_context, execution_context):
+def add_omega_constraint(model_context, execution_context, sex_id):
     """
     Adds a constraint to other-cause mortality rate. Removes mtother,
-    mtall, and mtspecific from observation data.
+    mtall, and mtspecific from observation data. Uses
+    :py:func:`cascade.input_data.configuration.builder.build_constraint` to make smoothing priors.
     """
-    MATHLOG.debug(f"Add omega constraint from age-standardized death rate data.")
     asdr = meas_bounds_to_stdev(
         age_groups_to_ranges(execution_context, get_age_standardized_death_rate_data(execution_context))
     )
     asdr["measure"] = "mtall"
     asdr = asdr.rename(columns={"location_id": "node_id"})
-    min_time = np.min(list(model_context.input_data.times))  # noqa: F841
-    max_time = np.max(list(model_context.input_data.times))  # noqa: F841
-    asdr = asdr.query("time_lower >= @min_time and time_upper <= @max_time and time_lower % 5 == 0")
+    asdr = asdr.query(f"sex_id == @sex_id")
+    if model_context.input_data.times:  # The times are a set so can be tested this way.
+        min_time = np.min(list(model_context.input_data.times))  # noqa: F841
+        max_time = np.max(list(model_context.input_data.times))  # noqa: F841
+        asdr = asdr.query("time_lower >= @min_time and time_upper <= @max_time and time_lower % 5 == 0")
```

Comment:
I know that this `%5` is to exclude the weird annual data points but that deserves a comment because it ain't obvious.